src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDaoImpl storageDao;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @Test
+    void register_user_isOK() {
+        User expected = new User("BobGood", "1234567", 18);
+        registrationService.register(expected);
+        assertEquals(expected, storageDao.get(expected.getLogin()));
+    }
+
+    @Test
+    void register_userPasswordIsShort_isNotOK() {
+        User expected = new User("BobWithShortPassword", "12", 18);
+        assertThrows(RegistrationException.class, () -> registrationService.register(expected));
```

Comment:
In your `register_userPasswordIsShort_isNotOK()` method, you are testing a password that is too short. However, you should also add a test for a password that is just short of the minimum length (5 characters), a password that is the exact minimum length (6 characters), and a password that is just over the minimum length (7 characters).